### PR TITLE
AADJ with via WHFB login

### PR DESCRIPTION
**Pull request by NagaCSC**
added AADJ with via WHFB login with Pin/Bio and then access on-premise resources

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/business/access-resources.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ Complete the following steps to enable your organization's Azure AD joined devic
 3. Once the Windows 10 devices are Azure AD joined, each user must reboot their devices and sign in with their Microsoft 365 Business credentials. All devices now have access to on-premises resources as well.
     
 No additional steps are required to get access to on-premises resources for Azure AD joined devices. This functionality is built into Windows 10. 
+
+If you have plans to login to the AADJ device other than password method Like PIN/Bio-metric via WHFB credential login and then access on-premise resources (shares,printers..etc), please follow https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base
```
Comment (JohanFreelancer9) [inline microsoft-365/business/access-resources.md:42]:
```suggestion
If you do not plan to log in to the AADJ device using the password method but would rather log in using another method like PIN/Bio-metric via WHFB credential login and then access on-premise resources (shares, printers, etc.), refer to [Configure Azure AD joined devices for On-premises Single-Sign On using Windows Hello for Business](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

File: microsoft-365/business/access-resources.md (hunk 1 above)
Comment (NagaCSC) [reply microsoft-365/business/access-resources.md:42]: That's sound good to me. Please proceed.

**Comment by mypil** [comment]
@NagaCSC  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Pull request merged**